gocourse09/zone/zone.go

```diff
@@ -0,0 +1,32 @@
+package zone
+
+type Specie int
+
+const (
+	Bull Specie = iota
+	Horse
+	Buffalo
+	Elk
+)
+
+type Animal struct {
+	Name   string
+	Specie Specie
+}
+
+type Zone interface {
+	GetAnimals() []Animal
+	AddAnimal(a Animal)
+}
+
+type FeedingZone struct {
+	Animals []Animal
```

Comment:
А для чого методи `GetAnimals` та `AddAnimals`, якщо клієнт може використати для маніпуляції експортоване поле `Animals`?

Напевно, треба зробити поле приватним?

```suggestion
	animals []Animal
```